src/app-bridge/app-bridge.ts

```diff
@@ -71,8 +71,20 @@ export type AppBridgeOptions = {
   initialLocale?: LocaleCode;
 };
 
+/**
+ * TODO: Consider validating locale if wrong code provided
+ */
+const getLocaleFromUrl = () =>
+  (new URL(window.location.href).searchParams.get("locale") as LocaleCode) || undefined;
+
+/**
+ * TODO: Probably remove empty string fallback
+ */
+const getDomainFromUrl = () => new URL(window.location.href).searchParams.get("domain") || "";
+
 const getDefaultOptions = (): AppBridgeOptions => ({
-  targetDomain: new URL(window.location.href).searchParams.get("domain") || "",
+  targetDomain: getDomainFromUrl(),
+  initialLocale: getLocaleFromUrl(),
```

Comment:
Fallback to EN is not required?

Reply:
AppBridgeState contains default state, including fallback to en